src/libasr/codegen/c_utils.h

```diff
@@ -906,7 +912,6 @@ class CCPPDSUtils {
             tmp_gen += indent + tab + "idx1 = step < 0 ? (idx1 >= s_len ? s_len-1 : idx1) : idx1;\n";
             tmp_gen += indent + tab + list_struct_type + " *__tmp = (" +
                     list_struct_type + "*) malloc(sizeof(" + list_struct_type + "));\n";
-            // tmp_gen += indent + tab + list_struct_type + " __tmp;\n";
```

Comment:
Should be removed?

Reply:
Yep.

Reply:
Remove it and then I will approve.

Reply:
That comment is unnecessary and we can remove it

Reply:
Have you pushed the changes? Its still showing up in the diff on Github.